tests/test_utils.py

```diff
@@ -0,0 +1,15 @@
+import sys
+from girder_worker.utils import TeeStdOutCustomWrite, TeeStdErrCustomWrite, JobManager
+
+
+def test_TeeStdOutCustomeWrite(capfd):
+    _nonlocal = {'data': ''}
+
+    def _append_to_data(message, **kwargs):
+        _nonlocal['data'] += message
+
+    with TeeStdOutCustomWrite(_append_to_data):
+        sys.stdout.write('Test String')
+        sys.stdout.flush()
+
+    assert _nonlocal['data'] == 'Test String'
```

Comment:
Can we also assert that the "real" stdout was written to by using the `capsys` fixture?

Reply:
sure,  though actually we should probably check the ```capfd``` fixture,  as i'm not sure how our hijacking and pytest's hijacking will mix.